Gemfile

```diff
@@ -14,7 +14,8 @@ gem 'bootsnap', '>= 1.1.0', require: false
 gem 'slim'
 
 group :development, :test do
-  gem 'byebug', platforms: [:mri, :mingw, :x64_mingw]
+  gem 'sqlite3', '1.3.13'
```

Comment:
The app is already using postgres, so we shouldn't need to add this. If it was added so the tests would run then we just needed to set up the test db.

Reply:
This was pretty dumb of me, huh?